### PR TITLE
feat(scrim): render default slot

### PR DESCRIPTION
**Pull request by asangma**
**Related Issue:** #2801

## Summary
* updates render to include default slot
* updates tests
* updates scrim bg color per @bstifle

### CODE REVIEW

File: src/components/calcite-scrim/calcite-scrim.scss [hunk 1]
```diff
@@ -25,7 +25,13 @@
   overflow-hidden
```
Comment (driskull) [inline src/components/calcite-scrim/calcite-scrim.scss:25]: Should the `select-none` be removed from scrim if we allow things inside it?

**Review by driskull — APPROVED** [review]

File: src/components/calcite-scrim/calcite-scrim.scss (hunk 1 above)
Comment (asangma) [reply src/components/calcite-scrim/calcite-scrim.scss:25]: That's a good point.
Right now, content in the default slot is still interact-able (in Chrome) and anything "under" the scrim remains non-interact-able.

I can add some tests.

Comment (asangma) [reply src/components/calcite-scrim/calcite-scrim.scss:25]: @driskull I added a couple tests to check for click. 

And wow...I totally misunderstood your suggestion.
Yes...we should remove `user-select: none`. Will do.

**Comment by asangma** [comment]
@ManahariDahal Would you mind having a look at the Scrim demo page and make sure I didn't break too much stuff?

**Comment by ManahariDahal** [comment]
> @ManahariDahal Would you mind having a look at the Scrim demo page and make sure I didn't break too much stuff?

looks good!

**Pull request merged**